pandas/tseries/tests/test_resample.py

```diff
@@ -1118,6 +1118,43 @@ def test_resample_loffset_count(self):
 
         assert_series_equal(result, expected)
 
+    def test_resample_loffset_agg(self):
```

Comment:
I would ideally like to move this test (and the loffset aboev) to the ``Base`` class as that fully tests DTI,PI, and TD indexes methodically (if that's too dififcult, then put the test method in *each* class), not in a single class.